src/test/java/net/openhft/chronicle/queue/RollCycleDefaultingTest.java

```diff
@@ -15,6 +16,11 @@ public void alias() {
         assertEquals(RollCycles.class, ObjectUtils.implementationToUse(RollCycle.class));
     }
 
+    @After
+    public void clearDefaultRollCycleProperty() {
+        System.clearProperty(QueueSystemProperties.DEFAULT_ROLL_CYCLE_PROPERTY);
+    }
```

Comment:
This was interfering with my test originally, I've since set the roll cycle explicitly, but felt like the right thing to do to clear it at the end.